### PR TITLE
Custom/aem initial setup

### PR DESCRIPTION
**Pull request by rakeshkumar-verma**
checked in project structure and incorporated all previous review comment.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+    <properties>
+        <aem.host>localhost</aem.host>
+        <aem.port>4502</aem.port>
+        <aem.publish.host>localhost</aem.publish.host>
+        <aem.publish.port>4503</aem.publish.port>
+        <sling.user>admin</sling.user>
+        <sling.password>admin</sling.password>
+        <vault.user>admin</vault.user>
+        <vault.password>admin</vault.password>
+        <core.wcm.components.version>2.7.0</core.wcm.components.version>
```
Comment (rishabh-behl) [inline pom.xml]: @rakeshkumar-verma : Use latest version of core component 2.8.1 https://github.com/adobe/aem-core-wcm-components

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
```
Comment (rishabh-behl) [inline pom.xml]: @rakeshkumar-verma we dont need this plugin as we are not using npm , pl delete 

frontend-maven-plugin

File: core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/*
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java]: @rakeshkumar-verma : why is this file required as part of project structure ? Wwe can add filters later when required.

File: core/src/main/java/com/hcl/ecomm/core/models/HelloWorldModel.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
+/*
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/models/HelloWorldModel.java:1]: @rakeshkumar-verma : Fine to have this as a placeholder file for checkin.

File: core/src/main/java/com/hcl/ecomm/core/servlets/SimpleServlet.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/servlets/SimpleServlet.java:1]: @rakeshkumar-verma : Fine to have this as a placeholder file for checkin.

File: core/src/test/java/com/hcl/ecomm/core/models/HelloWorldModelTest.java [hunk 6]
```diff
@@ -0,0 +1,63 @@
+/*
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/models/HelloWorldModelTest.java:1]: @rakeshkumar-verma : Fine to have this as a placeholder file for checkin.

File: core/src/test/java/com/hcl/ecomm/core/servlets/SimpleServletTest.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (rishabh-behl) [inline core/src/test/java/com/hcl/ecomm/core/servlets/SimpleServletTest.java:1]: @rakeshkumar-verma : Fine to have this as a placeholder file for checkin.

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<workspaceFilter version="1.0">
+    <filter root="/apps/hclecomm"/>
+    <filter root="/content/hclecomm" mode="merge"/>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/META-INF/vault/filter.xml:4]: @rakeshkumar-verma : <filter root="/content/hclecomm" mode="merge"/> /content/hclecomm this should not part of ui.apps filter xml. should go in content filter.xml

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/accordion/.content.xml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/accordion/.content.xml:1]: @rakeshkumar-verma : so u have checked in all component references too . Could have been avoided but ok for now.

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml [hunk 10]
```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml:1]: @rakeshkumar-verma : have u checked d that all thsese default policies are needed ?

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/summer/.content.xml [hunk 11]
```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/summer/.content.xml]: @rakeshkumar-verma : why doe we need this segment ?

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/winter/.content.xml [hunk 12]
```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/winter/.content.xml]: @rakeshkumar-verma : why do we need this segment ?

**Review by rishabh-behl — COMMENTED** [review]
@rakeshkumar-verma @DeepaliAgrwl : pl incorporate comments

File: pom.xml (hunk 1 above)
Comment (rakeshkumar-verma) [reply pom.xml]: updated core component version as 2.8.0

File: pom.xml (hunk 2 above)
Comment (rakeshkumar-verma) [reply pom.xml]: removed plugin.

File: core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java (hunk 3 above)
Comment (rakeshkumar-verma) [reply core/src/main/java/com/hcl/ecomm/core/filters/LoggingFilter.java]: removed.

File: ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 8 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/META-INF/vault/filter.xml:4]: removed content filter from ui.apps

File: core/src/main/java/com/hcl/ecomm/core/models/HelloWorldModel.java (hunk 4 above)
Comment (rakeshkumar-verma) [reply core/src/main/java/com/hcl/ecomm/core/models/HelloWorldModel.java:1]: Will remove once we have project specific code

File: core/src/main/java/com/hcl/ecomm/core/servlets/SimpleServlet.java (hunk 5 above)
Comment (rakeshkumar-verma) [reply core/src/main/java/com/hcl/ecomm/core/servlets/SimpleServlet.java:1]: Will remove once we have project specific code

File: core/src/test/java/com/hcl/ecomm/core/models/HelloWorldModelTest.java (hunk 6 above)
Comment (rakeshkumar-verma) [reply core/src/test/java/com/hcl/ecomm/core/models/HelloWorldModelTest.java:1]: Will remove once we have project specific code

File: core/src/test/java/com/hcl/ecomm/core/servlets/SimpleServletTest.java (hunk 7 above)
Comment (rakeshkumar-verma) [reply core/src/test/java/com/hcl/ecomm/core/servlets/SimpleServletTest.java:1]: Will remove once we have project specific code

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/accordion/.content.xml (hunk 9 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/accordion/.content.xml:1]: Will remove unused file once we have project specific code

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/summer/.content.xml (hunk 11 above)
Comment (rakeshkumar-verma) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/summer/.content.xml]: deleted.

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/winter/.content.xml (hunk 12 above)
Comment (rakeshkumar-verma) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/segments/winter/.content.xml]: deleted

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml (hunk 10 above)
Comment (rakeshkumar-verma) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml:1]: keeping default policy for reference and will remove once develop project specific

**Pull request merged**